src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -97,19 +96,39 @@ public async Task SendAsync_SlowServerAndCancel_ThrowsTaskCanceledException()
             }
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/20675")]
         [OuterLoop]
         [Fact]
-        [OuterLoop]
-        public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpRequestException()
+        public async void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpRequestException()
         {
             var handler = new WinHttpHandler();
             using (var client = new HttpClient(handler))
             {
-                Task<HttpResponseMessage> t = client.GetAsync(SlowServer);
+                var triggerResponseWrite = new TaskCompletionSource<bool>();
+                var triggerRequestWait = new TaskCompletionSource<bool>();
 
-                AggregateException ag = Assert.Throws<AggregateException>(() => t.Wait());
-                Assert.IsType<HttpRequestException>(ag.InnerException);
+                await LoopbackServer.CreateServerAsync(async (server, url) =>
+                {
+                    Task serverTask = server.AcceptConnectionAsync(async connection =>
+                    {
+                        await connection.SendResponseAsync($"HTTP/1.1 200 OK\r\nContent-Length: 16000\r\n\r\n");
+
+                        triggerRequestWait.SetResult(true);
+                        await triggerResponseWrite.Task;
+                    });
+
+                    HttpRequestException ex = await Assert.ThrowsAsync<HttpRequestException>(async () =>
+                    {
+                        Task<HttpResponseMessage> t = client.GetAsync(url);
+                        await triggerRequestWait.Task;
+                        var message = await t;
+                        var _ = message.Content;
```

Comment:
Is this needed? Isn't `GetAsync` creating and buffering the content?

Reply:
Apparently not.